blindsign/blindrsa/blindrsa.go

```diff
@@ -165,6 +184,17 @@ func (state RSAVerifierState) Finalize(data []byte) ([]byte, error) {
 	return sig, nil
 }
 
+// CopyBlind returns an encoding of the blind value used in the protocol.
+func (state RSAVerifierState) CopyBlind() []byte {
+	r := new(big.Int).ModInverse(state.rInv, state.verifier.pk.N)
+	return r.Bytes()
+}
+
+// CopySalt returns an encoding of the per-message salt used in the protocol.
+func (state RSAVerifierState) CopySalt() []byte {
+	return state.salt
```

Comment:
it needs to do an actual copy, otherwise `state.salt` can be modified externally.

Reply:
Good catch -- fixed!